packages/kotti-ui/source/kotti-field-file-upload/components/FileItemRemote.vue

```diff
@@ -1,5 +1,9 @@
 <template>
```

Comment:
Line `84` (`? translations.value.statusMsg[status.value]`) may need to be updated. It is indexing `statusMsg` in the translation obj but there's no entry for the new status `HIDDEN`. In theory, it shouldn't happen as these items are not rendered but TS is complaining (maybe pipelines too)